### DISCUSSION TITLE
Unable to enter SPF record

### DISCUSSION THREAD

**Discussion by astrabite** [vercel/community#3975] [Help]
### Summary

while entering the SPF records for my google workspace, i am gettting the below error :-

"The specified value contains non ASCII characters."

### Steps to Reproduce

![image](https://github.com/vercel/community/assets/144422330/4b2ab290-e9e7-49f3-ad15-2fc127b3afb8)

**Comment by amyegan** [comment]
Hey @astrabite. I'm not able to replicate this problem. Is is still happening for you with value `v=spf1 include:_spf.google.com ~all`?
